libs/admin/src/lib/services/admin.service.ts

```diff
@@ -55,10 +65,18 @@ export class AdminService {
     return this._httpClient.get('/api/admin/user').pipe(map((res) => res as User[]));
   }
 
+  public getUserNew(id: string): Observable<User> {
```

Comment:
Why is there a `new` prefix on these methods? Can't we just replace the old methods now?